media/css/firefox/new/trailhead/download.scss

```diff
@@ -82,7 +86,7 @@ $image-path: '/media/protocol/img';
 
         .mzp-c-hero-title {
             @include bidi(((background-position, top left, top right),));
-            max-width: 9em;
+            // max-width: 9em;
```

Comment:
I need to make this conditional on the en-US trailhead pages (AFAICT it is just to make the heading wrap nicely in English)

Reply:
Noting todo 👍